src/components/ProjectList/ProjectItem.js

```diff
@@ -40,98 +58,73 @@ class ProjectItem extends Component {
   render() {
     const {
       item,
-      childrenProp,
       primaryTextProp,
       secondaryTextProp,
       valueProp,
       clickHandler,
+      updated,
+      owner,
       inset,
     } = this.props
 
     const primaryText = item[primaryTextProp]
     const secondaryText = secondaryTextProp ? item[secondaryTextProp] : ''
     const value = item[valueProp]
-    const children = item[childrenProp]
-
-    const listItemWithNestedStyle = {
-      fontSize: '13px',
-    }
+    const ownerName = item[owner].first_name + ' ' + item.owner.last_name // eslint-disable-line
```

Comment:
Please, replace here with the `full_name`, as long as it's the same, and maybe simply we'll get rid of first and last.